src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -99,7 +98,7 @@ public int getWidth() {
 	 * @return the height of the field.
 	 */
 	public int getHeigth() {
-		return WINDOW_Y;
+		return (int) settings.getDouble("SCREEN_HEIGHT") - 50;
```

Comment:
There are other methods that want a double value, only this one wants an integer for the screen size..

<review